dags/public-health/covid19/sync-covid-testing-data.py

```diff
@@ -17,7 +17,12 @@ def get_data(filename, workbook, sheet_name):
     df.reset_index(level=0, inplace=True)
     df.columns = ["Date", "Performed", "Test Kit Inventory"]
     df["Date"] = pd.to_datetime(df["Date"], errors="coerce")
-    df = df.loc[df["Date"].dt.date < datetime.datetime.now().date()]
+    df = df.loc[
+        df["Date"].dt.date
```

Comment:
@ian-r-rose Is this conditional needed, or is it clearer to normalize everything to UTC, then right before overwriting, change it back to PST?

Reply:
@tiffanychu90 I agree it's a bit confusing, but since the layer in question is already in PST (if I understand correctly), then we may as well move forward with this.